js/index.js

```diff
@@ -0,0 +1,38 @@
+function render(pattern, getColor, scale = 100) {
+	const logo = document.getElementById('rs-logo');
+	logo.style.display = 'none';
+
+	const canvas = document.querySelector("canvas");
+	const ctx = canvas.getContext("2d");
+
+	const width = pattern[0].length;
+	const height = pattern.length;
+
+	canvas.width = width * scale;
+	canvas.height = height * scale;
+
+	for (let row = 0; row < height; row += 1) {
+		for (let col = 0; col < width; col += 1) {
+			ctx.fillStyle = getColor(pattern[row][col]);
+			ctx.fillRect(col * scale, row * scale, scale, scale);
+		}
+	}
+
+	canvas.style.display = 'block';
```

Comment:
The same as before, it will be better to just a remove class with display:none property instead of rewriting style.display value.